connection.c

```diff
@@ -43,6 +43,7 @@ typedef struct ConnCacheEntry
 	/* Remaining fields are invalid when conn is NULL: */
 	int			xact_depth;		/* 0 = no xact open, 1 = main xact open, 2 =
 								 * one level of subxact open, etc */
+	bool		updatable;		/* false for readonly connections */
```

Comment:
The change of `connection.c` is no longer necessary.

Reply:
Yes, this variable isn't necessary by algorithm, but in perspective of a diagnostic function `sqlite_fdw_server_options(servername text)` like https://github.com/ibarwick/firebird_fdw#functions this variable will be usefully. Must I remove this variable in this PR anyway?

Reply:
It depends on the implementation of sqlite_fdw_server_options. Could you remove this variable in this PR because it is unrelated change for this PR?

Reply:
Ok, @t-kataym , I will remove this variable.

Reply:
@bichht0608 and @t-kataym , I have successfully tests for code after https://github.com/pgspider/sqlite_fdw/pull/59/commits/ea4b15fa7fb6bf13aa0f8d7b7783ccb2696687b8, please verify. Look like this PR comes to finish.

Reply:
Thank you for fixing.